docusaurus/docs/cms/features/preview.md

```diff
@@ -301,10 +301,16 @@ To track this, within your front-end application, add an event listener to liste
 
 With Next.js, the recommended way to refresh the iframe content is with <ExternalLink to="https://nextjs.org/docs/app/building-your-application/caching#routerrefresh" text="the `router.refresh()` method" />.
 
+The implementation slighltly differs depending on which feature is implemented, Preview available in the Free plan or Live Preview available in the <GrowthBadge/> and <EnterpriseBadge/> plans:
```

Comment:
I would rearrange things a bit. The Preview part (CE) does not need the handleMessage useEffect at all, because strapiUpdate only gets triggered from the side editor or the input popovers, and without EE they don't have access to either.

So it's a common CE setup (building the URL, fetching draft content if needed). Then the handleMessage part and the source maps header are only in order to power the Live Preview functionality

Reply:
I'm happy to, but in this case I'm suggesting to rearrange the structure of the page a bit, so it's not ideal

Instead of using tabs for Preview / Live Preview, show the Preview (without the useEffect snippet) part in a h2, as these steps will be common for all users.

Then have another h2 for Live Preview with the EE badges. In this section there would be the useEffect snippet and the encode-source-maps header